cmd/bundle/main.go

```diff
@@ -0,0 +1,124 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/docker/cli/cli/config"
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
+	"github.com/shipwright-io/build/pkg/reconciler/buildrun/resources/sources"
+	"github.com/spf13/pflag"
+)
+
+var flagValues struct {
+	image      string
+	target     string
+	secretPath string
+}
+
+func init() {
+	pflag.StringVar(&flagValues.image, "image", "", "Location of the bundle image (mandatory)")
+	pflag.StringVar(&flagValues.target, "target", "/workspace/source", "The target directory to place the code")
+	pflag.StringVar(&flagValues.secretPath, "secret-path", "", "A directory that contains access credentials (optional)")
+}
+
+func main() {
+	if err := Do(context.Background()); err != nil {
+		log.Fatal(err.Error())
```

Comment:
I guess we here must return with a non-zero status code to fail the build ?

Reply:
The `log.Fatal` function does an `os.Exit(!=0)` after logging the line.